flipperdevices/flipperzero-firmware
# fixed line end in cli:hw_info command

# What's new

- Changed "\r" to "\r\n" inside hw_info command

# Verification 

- connect to CLI
- type "hw_info"
- Check, that newline between "Production date" and "Name" correct.

# Checklist (do not modify)
